deployment/ccip/deploy.go

```diff
@@ -125,6 +125,136 @@ func deployContract[C Contracts](
 	return &contractDeploy, nil
 }
 
+// DeployPrerequisiteContracts deploys the contracts that can be ported from previous CCIP version to the new one.
+// This is only required for staging and test environments where the contracts are not already deployed.
+// We are not deploying router and RMNProxy here as RMNProxy has dependency on RMNRemote which is a new contract and
```

Comment:
Wait I thought we were going to deploy router + RMNProxy pointing at MockRMN to simulate v1? Then separate RMNProxy for V2 until we adjust RMNRemote

Reply:
Yes, I got confused here. Changing